packages/core/src/components/TextField/TextField.test.jsx

```diff
@@ -90,44 +91,59 @@ describe('TextField', function() {
   });
 
   it('has a label', () => {
-    const data = shallowRender();
+    const data = render();
     const label = data.wrapper.find('FormLabel').first();
 
     expect(label.prop('children')).toBe(data.props.label);
   });
 
   it('has a hint', () => {
-    const data = shallowRender({ hint: '123' });
+    const data = render({ hint: '123' });
     const label = data.wrapper.find('FormLabel').first();
 
     expect(label.prop('hint')).toBe(data.props.hint);
   });
 
   it('adds className to root element', () => {
-    const data = shallowRender({ className: 'bar' });
+    const data = render({ className: 'bar' });
 
     expect(data.wrapper.hasClass('bar')).toBe(true);
   });
 
   it('adds className to field', () => {
-    const data = shallowRender({ fieldClassName: 'bar' });
+    const data = render({ fieldClassName: 'bar' });
     const field = data.wrapper.find('.ds-c-field').first();
 
     expect(field.hasClass('ds-c-field')).toBe(true);
     expect(field.hasClass('bar')).toBe(true);
   });
 
   it('adds className to label', () => {
-    const data = shallowRender({ labelClassName: 'bar' });
+    const data = render({ labelClassName: 'bar' });
 
     expect(data.wrapper.find('FormLabel').hasClass('bar')).toBe(true);
   });
 
+  it('returns reference to input field', () => {
+    let ref;
+    const data = render(
+      {
+        defaultValue: 'Yay',
+        fieldRef: el => {
+          ref = el;
```

Comment:
@hannaliebl Here's how I got this to work. This is similar to how this prop would be used within a real world component, ie:

```jsx
<TextField fieldRef={el => { this.input = el } } />
```